test/test_ppca.py

```diff
@@ -55,7 +47,7 @@ def test_probabilistic_pca_multi_device(
     train_loader = torch.utils.data.DataLoader(
         TestDataset(x_ng),
         batch_size=batch_size,
-        shuffle=True,
+        shuffle=False,
```

Comment:
PyTorch's sampler behaves strangely with `devices=2`. Disabling shuffling resolves the issue and PPCA achieves high accuracy fit.